paddle/fluid/framework/ir/mkldnn/requant_mkldnn_fuse_pass.cc

```diff
@@ -0,0 +1,146 @@
+            const int size = scales_vector.size();
+            
+            Tensor tensor;
+            tensor.mutable_data<float>({size}, platform::CPUPlace());
+            auto dev_ctx = paddle::platform::CPUDeviceContext();
+            TensorFromVector(scales_vector, dev_ctx, &tensor);
+            var_quant_scales[name] = std::make_pair(0, &tensor);
+            op_desc->RemoveAttr(fake_name);
+          }
+        }
+        break;
+      }
+    }
+  }
+}
+
+// void RequantMkldnnFusePass::ComputeWeightScales(ir::Graph* graph, Scope* scope,
+//   std::unordered_map<std::string, Tensor*>& weight_thresholds,
+//   std::unordered_map<std::string, std::pair<int, Tensor*>>& var_quant_scales) const {
+//   // auto array_max = [&](Tensor* tensor, int axis) {
+//   //    auto data = tensor->mutable_data<float>(platform::CPUPlace());
+//   //    const auto dims = tensor->dims();
+//   //    PADDLE_ENFORCE_EQ(dims.size(), 2,
+//   //       "The input persistable var of op is not found.");
+//   // };
+  
+//   auto compute_var_scales = [&](ir::Graph* graph, 
+//     std::unordered_set<std::string> ops, 
+//     std::string weight_name, int axis) {
+//       for (auto* op_node :
+//        ir::TopologyVarientSort(*graph, static_cast<ir::SortKind>(0))) {
+//         if (!op_node->IsOp()) continue;
+
+//         auto* op_desc = op_node->Op();
+//         if (ops.count(op_desc->Type())) {
+//           auto var_name = op_desc->Input(weight_name)[0];
+//           auto* var = scope->FindVar(var_name);
+//           PADDLE_ENFORCE_NOT_NULL(var,
+//                             "The input persistable var of %s op is not found.",
+//                             op_desc->Type());
+//           auto* weight_tensor = var->GetMutable<LoDTensor>();
+//           const auto dims = weight_tensor->dims();
+//           int volume = 1;
+//           for (int i = 1; i < dims.size(); i++) {
+//             volume *= dims[i];
+//           }
+//           std::vector<int64_t> reshape_dims = {dims[0], volume};
+//           weight_tensor->Resize(framework::make_ddim(reshape_dims));
+//           auto weight_data = weight_tensor->mutable_data<float>(platform::CPUPlace());
+//           for (int i = 0; i < weight_tensor->numel(); i++) {
+//             weight_data[i] = fabs(weight_data[i]);
+//           }
+//         }
+//       }
+//   };
+// }
+
+void RequantMkldnnFusePass::ApplyImpl(ir::Graph* graph) const {
+  VLOG(3) << "Convert paddle slim quantized model to mkldnn quantized model.";
+  const std::string pattern_name = "requant_mkldnn_fuse_pass";
+  FusePassBase::Init(pattern_name, graph);
+  
+  std::unordered_map<std::string, Tensor*> weight_thresholds;
+  std::unordered_map<std::string, std::pair<int, Tensor*>> var_quant_scales;
+
+  auto* scope = param_scope();
+  GetQuantInfo(graph, scope, weight_thresholds, var_quant_scales);
+  
+  //ComputeWeightScales(graph, scope);
```

Comment:
I understand that this pass will functionally correspond to this `graph = self._compute_weight_scales (graph)`, do you also plan to add the functionality of  `graph = self._propagate_scales (graph)` here?